rgdc/rgdc/rgdc.py

```diff
@@ -183,7 +183,6 @@ def search(
         resolution: Optional[Tuple[int, int]] = None,
         cloud_cover: Optional[Tuple[float, float]] = None,
         frame_rate: Optional[Tuple[int, int]] = None,
-        limit: Optional[int] = None,
         offset: Optional[int] = None,
```

Comment:
Limit and offset typically go together. If our intent is to disable all pagination capabilities, we should remove both of them.

Reply:
Ah thanks for catching that